packages/react-reconciler/src/ReactFiber.js

```diff
@@ -968,10 +960,7 @@ export function assignFiberPropertiesInDEV(
     target.selfBaseDuration = source.selfBaseDuration;
     target.treeBaseDuration = source.treeBaseDuration;
   }
-  if (enableUserTimingAPI) {
-    target._debugID = source._debugID;
-    target._debugIsCurrentlyTiming = source._debugIsCurrentlyTiming;
-  }
+  target._debugID = source._debugID;
```

Comment:
This should be in an `if (__DEV__)` block, I think.

Reply:
This whole function is DEV-only so I'm not sure this is necessary.

Reply:
I realize the function is named `assignFiberPropertiesInDEV` but it's not actually gated anywhere to be DEV only.

Reply:
It's gated at the usage site. But I agree it would make sense to wrap its body too.

Reply:
Yeah, we usually gate in both places I think.

Not new to this PR, just something I noticed while looking at the code changes.